src/pages-sections/transactions/add-transaction-form.jsx

```diff
@@ -0,0 +1,221 @@
+'use client';
+
+import React from 'react';
+import Card from '@mui/material/Card';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import { Formik } from 'formik';
+import * as yup from 'yup'; // GLOBAL CUSTOM COMPONENTS
+import PropTypes from 'prop-types';
+import { MenuItem } from '@mui/material';
+
+const VALIDATION_SCHEMA = yup.object().shape({
+  ebill: yup.number().required('Electricity bill is required'),
+  wbill: yup.number().required('Water bill is required'),
+  municipal_tax: yup.number().required('Municipal tax is required'),
+  bank_emi: yup.number().required('Bank emi is required'),
```

Comment:
Don't mix casing, stick with camelCase across the project. snake_case is used mainly in python